staging/src/k8s.io/apiserver/pkg/server/options/encryptionconfig/config_test.go

```diff
@@ -187,6 +187,8 @@ func TestLegacyConfig(t *testing.T) {
 
 func TestEncryptionProviderConfigCorrect(t *testing.T) {
 	defer featuregatetesting.SetFeatureGateDuringTest(t, utilfeature.DefaultFeatureGate, features.KMSv2, true)()
+	defer featuregatetesting.SetFeatureGateDuringTest(t, utilfeature.DefaultFeatureGate, features.RemoveKMSv1, false)()
```

Comment:
These should not be required.

Reply:
All the existing tests will fail if we don't set `KMSv1=true` as default is false.

Reply:
The default must be true.  Nothing should break (that comes when v2 goes GA).

Reply:
sounds good!